OpenKeychain/src/main/java/org/sufficientlysecure/keychain/securitytoken/GenerateKeyUseCase.java

```diff
@@ -0,0 +1,50 @@
+package org.sufficientlysecure.keychain.securitytoken;
+
+
+import java.io.IOException;
+
+import org.sufficientlysecure.keychain.util.Passphrase;
+
+
+public class GenerateKeyUseCase {
```

Comment:
Looks like this is not used anywhere

Reply:
It's not, never was iirc. I guess we left it in just to keep the code close :)